WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -1586,7 +1586,7 @@ function(startDate, duration, folderId, mailItem) {
 ZmCalViewController.prototype._newApptObject =
 function(startDate, duration, folderId, mailItem) {
 	var newAppt = new ZmAppt();
-	newAppt.setStartDate(AjxDateUtil.roundTimeMins(startDate, 30));
+	newAppt.setStartDate(AjxDateUtil.roundTimeMins(startDate, parseInt(appCtxt.get(ZmSetting.CAL_SCALING))));
```

Comment:
specify base as 10 for parseInt
required for older browsers

Reply:
Fixed.